linkerd/identifier/http/src/main/scala/io/l5d/BasicHttpIdentifierInitializer.scala

```diff
@@ -0,0 +1,26 @@
+package io.l5d.identifier
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.{Dtab, Path}
+import io.buoyant.linkerd.IdentifierInitializer
+import io.buoyant.router.http.Identifier
+import io.l5d.HttpIdentifierConfig
+
+class BasicHttpIdentifierInitializer extends IdentifierInitializer {
+  val configClass = classOf[BasicHttpIdentifierConfig]
+  override val configId = BasicHttpIdentifierConfig.kind
+}
+
+object BasicHttpIdentifierInitializer extends BasicHttpIdentifierInitializer
+
+object BasicHttpIdentifierConfig {
+  val kind = "default"
```

Comment:
tioli: I know before I suggested that this should be "default", but since we refer to this everywhere as _basic_, maybe it should just be that.  My main objection was to having a fully qualified classname.